libr/core/cbin.c

```diff
@@ -2181,9 +2191,13 @@ static int bin_sections(RCore *r, int mode, ut64 laddr, int va, ut64 at, const c
 	} else if (IS_MODE_RAD (mode) && !at) {
 		r_cons_printf ("fs %ss\n", type);
 	} else if (IS_MODE_NORMAL (mode) && !at && !printHere) {
-		r_cons_printf ("[%s]\n", print_segments ? "Segments" : "Sections");
+		r_cons_printf ("[%s]\n"
+                   "Nm Paddr       Size Vaddr      Memsz Perms%s Name\n",
```

Comment:
why is `chksum` printed only when `!printHere`? I think it could the checksum of the section/segment could be printed even when `printHere` is true.

Reply:
moreover, I think in the `header` you should print something more meaningful like `checksum` or something. If I understood it right `chksum` is just `.` or something like that, so it doesn't really explain that it's a checksum.

Reply:
So chksum is either "entropy" or "sha256" here, and I went with the lazy approach of passing that char * rather than having a chain of strcmp's. However this does have a couple of downsides, namely that the first letter doesn't get capitalized like the other headers, and if the user enters in some odd or unsupported value (e.g. MD5), no checksum will be printed but the value they entered is printed as a header. I can add additional logic to check if it is "entropy" or "sha256" and print the appropriate thing in those cases?

Reply:
And for the !printHere I hadn't looked in enough detail at what the existing if statement was doing. I will add a separate if statement for printing headers.

Reply:
Ok I double checked after what you told. And yeah... it's fine as you have done :) sorry about that.

Reply:
But you should put a space between `Perms` and `%s`

Reply:
I've now just set it to print 'Checksum' as you originally suggested, I think that looks neater than printing what the user typed with incorrect capitalization.